ui/src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Be consistent whether to use single quotes or double quotes.

Reply:
The Google JS guide prefers single so I went with that.